assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java

```diff
@@ -12692,10 +12694,12 @@ public void doSet_defaultNoSubmissionScore(RunData data) {
 
                 // Only record the default grade setting for no-submission if there were no errors produced
                 if (state.getAttribute(STATE_MESSAGE) == null) {
+                    String inputGrade = grade;
                     grade = scalePointGrade(state, grade, a.getScaleFactor());
```

Comment:
maybe this line is not even needed?

Reply:
The scaled grade is used later on. It could probably be moved, but was trying to make the changes minimal.

Reply:
I wouldn't worry about minimal changes! Readable and concise code is more important!

Reply:
Ok, I'll move it.